### ISSUE TITLE
Windows CI failure

### ISSUE THREAD

**Issue by yvt** [yvt/openspades#994]
<https://github.com/yvt/openspades/runs/5023601726?check_suite_focus=true#step:5:65>

```
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:184 (message):
      
      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "***your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  scripts/cmake/vcpkg_acquire_msys.cmake:90 (vcpkg_download_distfile)
  scripts/cmake/vcpkg_acquire_msys.cmake:138 (msys_package_download)
  scripts/cmake/vcpkg_find_acquire_program.cmake:446 (vcpkg_acquire_msys)
  scripts/cmake/vcpkg_fixup_pkgconfig.cmake:112 (vcpkg_find_acquire_program)
  ports/brotli/portfile.cmake:25 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:136 (include)


Error: Building package brotli:x86-windows-static failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: brotli:x86-windows-static
  Vcpkg version: 2020.11.12-nohash
```

**Comment by VilotStar** [comment]
Found anything out about this yet?

**Comment by yvt** [comment]
Updating the `vcpkg` submodule to a newer version will probably solve the issue. Just in case, we also need someone having access to a real Windows machine to test some basic functionalities (e.g., whether the sounds play correctly) after the update.

**Comment by nonperforming** [comment]
I can reproduce on my local Win10 machine.
Before updating vcpkg, Brotli fails to build (dependency of freetype) because it was missing MSYS2 to build and could not source the files from numerous URL's (even though it was installed)

Updating vcpkg seems to work... 
Still compiling OpenGL, I'll leave this installing dependencies and see if it succeeds.

**Comment by nonperforming** [comment]
Error: may just be my inexperience with CMake, or due to me using VS2022.
```
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.

Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19043.
CMake Error at CMakeLists.txt:2 (project):
  Failed to run MSBuild command:

    C:/Program Files/Microsoft Visual Studio/2022/Community/MSBuild/Current/Bin/amd64/MSBuild.exe

  to get the value of VCTargetsPath:

    Microsoft (R) Build Engine version 17.0.0+c9eb9dd64 for .NET Framework

    Copyright (C) Microsoft Corporation. All rights reserved.

    Build started 19/02/2022 1:47:23 am.

    Project "C:\Users\itsaw\Documents\Git\openspades\openspades.msvc\CMakeFiles\3.22.1\VCTargetsPath.vcxproj" on node 1 (default targets).

    C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppBuild.targets(436,5): error MSB8020: The build tools for C:/Users/itsaw/Documents/Git/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake (Platform Toolset = 'C:/Users/itsaw/Documents/Git/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake') cannot be found. To build using the C:/Users/itsaw/Documents/Git/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake build tools, please install C:/Users/itsaw/Documents/Git/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake build tools.  Alternatively, you may upgrade to the current Visual Studio tools by selecting the Project menu or right-click the solution, and then selecting "Retarget solution". [C:\Users\itsaw\Documents\Git\openspades\openspades.msvc\CMakeFiles\3.22.1\VCTargetsPath.vcxproj]

    Done Building Project "C:\Users\itsaw\Documents\Git\openspades\openspades.msvc\CMakeFiles\3.22.1\VCTargetsPath.vcxproj" (default targets) -- FAILED.

    Build FAILED.

    "C:\Users\itsaw\Documents\Git\openspades\openspades.msvc\CMakeFiles\3.22.1\VCTargetsPath.vcxproj" (default target) (1) ->

    (PrepareForBuild target) -> 

      C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppBuild.targets(436,5): error MSB8020: The build tools for C:/Users/itsaw/Documents/Git/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake (Platform Toolset = 'C:/Users/itsaw/Documents/Git/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake') cannot be found. To build using the C:/Users/itsaw/Documents/Git/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake build tools, please install C:/Users/itsaw/Documents/Git/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake build tools.  Alternatively, you may upgrade to the current Visual Studio tools by selecting the Project menu or right-click the solution, and then selecting "Retarget solution". [C:\Users\itsaw\Documents\Git\openspades\openspades.msvc\CMakeFiles\3.22.1\VCTargetsPath.vcxproj]

        0 Warning(s)

        1 Error(s)

    Time Elapsed 00:00:00.48

  Exit code: 1

Configuring incomplete, errors occurred!
See also "C:/Users/itsaw/Documents/Git/openspades/openspades.msvc/CMakeFiles/CMakeOutput.log".
```

[CMakeCache.txt](https://github.com/yvt/openspades/files/8097239/CMakeCache.txt)
[CMakeOutput.log](https://github.com/yvt/openspades/files/8097238/CMakeOutput.log)

**Comment by Conticop** [comment]
Built d7d467e70954b6de080823c600b461ddeccf5e09 on Windows 10 just fine. OpenAL and YSR Audio are both working too.

**Comment by nonperforming** [comment]
Building using @Conticop's powershell script worked flawlessly.

**Comment by yvt** [comment]
Fixed by #1017
